src/scripts/main.js

```diff
@@ -1,3 +1,79 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const appendRowButton = document.querySelector('.append-row');
+  const removeRowButton = document.querySelector('.remove-row');
+  const appendColumnButton = document.querySelector('.append-column');
+  const removeColumnButton = document.querySelector('.remove-column');
+  const table = document.querySelector('table');
+  const tableBody = table.querySelector('tbody');
+
+  const maxRows = 10;
+  const minRows = 2;
+  const maxColumns = 10;
+  const minColumns = 2;
+
+  function updateButtons() {
+    const rowCount = tableBody.rows.length;
+    const columnCount = tableBody.rows[0].cells.length;
```

Comment:
You have repeated this logic in multiple places. Consider refactoring your code to avoid duplication. You could create a function that returns the current row and column counts.